html-tables/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head> <title>HTML table</title>
+</head>
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+    <tr>
```

Comment:
just make sure to add the `thead` element for the first table row.

Reply:
added it in and pushed though